### PR TITLE
Andy edits

### PR DESCRIPTION
**Pull request by aterrel**
A few minor edits.

### CODE REVIEW

File: simplification.tex [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 The generic way to simplify an expression is by calling the \texttt{simplify}
 function.
-It must be emphasized that simplification is not an unambiguously defined
+It must be emphasized that simplification is not a rigously defined
```
Comment (sumith1896) [inline simplification.tex]: Typo

**Comment by asmeurer** [comment]
Can you add your information to the authors file?

**Comment by aterrel** [comment]
Sure I just didn't know which order I was in.

**Comment by asmeurer** [comment]
The prettify script will fix the order. You just have to pull in your commit number from the sympy_shortlog.txt.

**Comment by asmeurer** [comment]
And I'll let @certik verify your authors info. I am +1 to the changes to the paper.

**Comment by asmeurer** [comment]
Oh and apparently there are merge conflicts.

**Comment by aterrel** [comment]
fixed the merge conflict

**Comment by asmeurer** [comment]
Merging this now to avoid any potential merge conflicts. @certik let Andy know if he needs to do anything else to be listed as an author.

**Comment by certik** [comment]
Here is authors.md as of 5bd87a8befcfba6d2fe56ccbb4d9c92420d8359f:

# Article Authors

All authors satisfy the following ICMJE [criteria](http://www.icmje.org/recommendations/browse/roles-and-responsibilities/defining-the-role-of-authors-and-contributors.html#two) for authorship:
1. Substantial contributions to the conception or design of the work; or the
   acquisition, analysis, or interpretation of data for the work; AND
2. Drafting the work or revising it critically for important intellectual
   content; AND
3. Final approval of the version to be published; AND
4. Agreement to be accountable for all aspects of the work in ensuring that
   questions related to the accuracy or integrity of any part of the work are
   appropriately investigated and resolved.

The column `SymPy Commits` lists how many commits the authors contributed to
SymPy (with a link to the actual commits), which shows how the requirement 1.
is satisfied.

The column `Article Contributions` lists all commits and comments to the
article itself, which shows how the requirement 2. is satisfied.

All authors approved the final version of the article, satisfying the
requirement 3.

Finally, all authors agreed to the requirement 4.

| Author | GitHub Profile | SymPy Commits | Article Contributions | Affiliation |
| :-- | :-- | :-: | :-- | :-- |
| Aaron Meurer | [@asmeurer](https://github.com/asmeurer) | [2639](https://github.com/sympy/sympy/commits?author=asmeurer) | [commits](https://github.com/sympy/sympy-paper/commits?author=asmeurer), [comments](https://github.com/sympy/sympy-paper/search?q=commenter:asmeurer&type=Issues) | University of South Carolina, Columbia, SC 29201 |
| Christopher P. Smith | [@smichr](https://github.com/smichr) | [4011](https://github.com/sympy/sympy/commits?author=smichr) | [commits](https://github.com/sympy/sympy-paper/commits?author=smichr), [comments](https://github.com/sympy/sympy-paper/search?q=commenter:smichr&type=Issues) | Polar Semiconductor, Inc., Bloomington, MN 55425 |
| Mateusz Paprocki | [@mattpap](https://github.com/mattpap) | [1450](https://github.com/sympy/sympy/commits?author=mattpap) | [commits](https://github.com/sympy/sympy-paper/commits?author=mattpap), [comments](https://github.com/sympy/sympy-paper/search?q=commenter:mattpap&type=Issues) | Continuum Analytics, Inc., Austin, TX 78701 |
| Ondřej Čertík | [@certik](https://github.com/certik) | [1443](https://github.com/sympy/sympy/commits?author=certik) | [commits](https://github.com/sympy/sympy-paper/commits?author=certik), [comments](https://github.com/sympy/sympy-paper/search?q=commenter:certik&type=Issues) | Los Alamos National Laboratory, Los Alamos, NM 87545 |
| Sergey B. Kirpichev | [@skirpichev](https://github.com/skirpichev) | [1055](https://github.com/sympy/sympy/commits?author=skirpichev) | [commits](https://github.com/sympy/sympy-paper/commits?author=skirpichev), [comments](https://github.com/sympy/sympy-paper/search?q=commenter:skirpichev&type=Issues) | Moscow State University, Faculty of Physics, Leninskie Gory, Moscow, 119991, Russia |
| Matthew Rocklin | [@mrocklin](https://github.com/mrocklin) | [1046](https://github.com/sympy/sympy/commits?author=mrocklin) | [commits](https://github.com/sympy/sympy-paper/commits?author=mrocklin), [comments](https://github.com/sympy/sympy-paper/search?q=commenter:mrocklin&type=Issues) | Continuum Analytics, Inc., Austin, TX 78701 |
| AMiT Kumar | [@aktech](https://github.com/aktech) | [263](https://github.com/sympy/sympy/commits?author=aktech) | [commits](https://github.com/sympy/sympy-paper/commits?author=aktech), [comments](https://github.com/sympy/sympy-paper/search?q=commenter:aktech&type=Issues) | Delhi Technological University, Shahbad Daulatpur, Bawana Road, New Delhi 110042, India |
| Sergiu Ivanov | [@scolobb](https://github.com/scolobb) | [260](https://github.com/sympy/sympy/commits?author=scolobb) | [commits](https://github.com/sympy/sympy-paper/commits?author=scolobb), [comments](https://github.com/sympy/sympy-paper/search?q=commenter:scolobb&type=Issues) | Université Paris Est Créteil, 61 av. Général de Gaulle, 94010 Créteil, France |
| Jason K. Moore | [@moorepants](https://github.com/moorepants) | [255](https://github.com/sympy/sympy/commits?author=moorepants) | [commits](https://github.com/sympy/sympy-paper/commits?author=moorepants), [comments](https://github.com/sympy/sympy-paper/search?q=commenter:moorepants&type=Issues) | University of California, Davis, Davis, CA 95616 |
| Sartaj Singh | [@leosartaj](https://github.com/leosartaj) | [200](https://github.com/sympy/sympy/commits?author=leosartaj) | [commits](https://github.com/sympy/sympy-paper/commits?author=leosartaj), [comments](https://github.com/sympy/sympy-paper/search?q=commenter:leosartaj&type=Issues) | Indian Institute of Technology (BHU), Varanasi, Uttar Pradesh 221005, India |
| Thilina Rathnayake | [@thilinarmtb](https://github.com/thilinarmtb) | [186](https://github.com/sympy/sympy/commits?author=thilinarmtb) | [commits](https://github.com/sympy/sympy-paper/commits?author=thilinarmtb), [comments](https://github.com/sympy/sympy-paper/search?q=commenter:thilinarmtb&type=Issues) | University of Moratuwa, Bandaranayake Mawatha, Katubedda, Moratuwa 10400, Sri Lanka |
| Sean Vig | [@flacjacket](https://github.com/flacjacket) | [186](https://github.com/sympy/sympy/commits?author=flacjacket) | [commits](https://github.com/sympy/sympy-paper/commits?author=flacjacket), [comments](https://github.com/sympy/sympy-paper/search?q=commenter:flacjacket&type=Issues) | University of Illinois at Urbana-Champaign, Urbana, IL 61801 |
| Brian E. Granger | [@ellisonbg](https://github.com/ellisonbg) | [160](https://github.com/sympy/sympy/commits?author=ellisonbg) | [commits](https://github.com/sympy/sympy-paper/commits?author=ellisonbg), [comments](https://github.com/sympy/sympy-paper/search?q=commenter:ellisonbg&type=Issues) | California Polytechnic State University, San Luis Obispo, CA 93407 |
| Richard P. Muller | [@rpmuller](https://github.com/rpmuller) | [158](https://github.com/sympy/sympy/commits?author=rpmuller) | [commits](https://github.com/sympy/sympy-paper/commits?author=rpmuller), [comments](https://github.com/sympy/sympy-paper/search?q=commenter:rpmuller&type=Issues) | Center for Computing Research, Sandia National Laboratories, Albuquerque, NM 87185 |
| Francesco Bonazzi | [@Upabjojr](https://github.com/Upabjojr) | [156](https://github.com/sympy/sympy/commits?author=Upabjojr) | [commits](https://github.com/sympy/sympy-paper/commits?author=Upabjojr), [comments](https://github.com/sympy/sympy-paper/search?q=commenter:Upabjojr&type=Issues) | Max Planck Institute of Colloids and Interfaces, Department of Theory and Bio-Systems, Am Mühlenberg 1, 14424 Potsdam, Germany |
| Harsh Gupta | [@hargup](https://github.com/hargup) | [123](https://github.com/sympy/sympy/commits?author=hargup) | [commits](https://github.com/sympy/sympy-paper/commits?author=hargup), [comments](https://github.com/sympy/sympy-paper/search?q=commenter:hargup&type=Issues) | Indian Institute of Technology Kharagpur, Kharagpur, West Bengal 721302, India |
| Shivam Vats | [@shivamvats](https://github.com/shivamvats) | [122](https://github.com/sympy/sympy/commits?author=shivamvats) | [commits](https://github.com/sympy/sympy-paper/commits?author=shivamvats), [comments](https://github.com/sympy/sympy-paper/search?q=commenter:shivamvats&type=Issues) | Indian Institute of Technology Kharagpur, Kharagpur, West Bengal 721302, India |
| Fredrik Johansson | [@fredrik-johansson](https://github.com/fredrik-johansson) | [119](https://github.com/sympy/sympy/commits?author=fredrik-johansson) | [commits](https://github.com/sympy/sympy-paper/commits?author=fredrik-johansson), [comments](https://github.com/sympy/sympy-paper/search?q=commenter:fredrik-johansson&type=Issues) | INRIA Bordeaux-Sud-Ouest -- LFANT project-team, 200 Avenue de la Vieille Tour, 33405 Talence, France |
| Fabian Pedregosa | [@fabianp](https://github.com/fabianp) | [114](https://github.com/sympy/sympy/commits?author=fabianp) | [commits](https://github.com/sympy/sympy-paper/commits?author=fabianp), [comments](https://github.com/sympy/sympy-paper/search?q=commenter:fabianp&type=Issues) | INRIA -- SIERRA project-team, 2 Rue Simone IFF, 75012 Paris, France |
| Matthew J. Curry | [@mattcurry](https://github.com/mattcurry) | [67](https://github.com/sympy/sympy/commits?author=mattjcurry@gmail.com) | [commits](https://github.com/sympy/sympy-paper/commits?author=mattcurry), [comments](https://github.com/sympy/sympy-paper/search?q=commenter:mattcurry&type=Issues) | Department of Physics and Astronomy, University of New Mexico, Albuquerque, NM 87131 |
| Andy R. Terrel | [@aterrel](https://github.com/aterrel) | [46](https://github.com/sympy/sympy/commits?author=aterrel) | [commits](https://github.com/sympy/sympy-paper/commits?author=aterrel), [comments](https://github.com/sympy/sympy-paper/search?q=commenter:aterrel&type=Issues) | Fashion Metric, Inc, Austin, TX 78681 |
| Štěpán Roučka | [@rouckas](https://github.com/rouckas) | [24](https://github.com/sympy/sympy/commits?author=rouckas) | [commits](https://github.com/sympy/sympy-paper/commits?author=rouckas), [comments](https://github.com/sympy/sympy-paper/search?q=commenter:rouckas&type=Issues) | Faculty of Mathematics and Physics, Charles University in Prague, V Holešovičkách 2, 180 00 Praha, Czech Republic |
| Ashutosh Saboo | [@ashutoshsaboo](https://github.com/ashutoshsaboo) | [17](https://github.com/sympy/sympy/commits?author=ashutoshsaboo) | [commits](https://github.com/sympy/sympy-paper/commits?author=ashutoshsaboo), [comments](https://github.com/sympy/sympy-paper/search?q=commenter:ashutoshsaboo&type=Issues) | Birla Institute of Technology and Science, Pilani, K.K. Birla Goa Campus, NH 17B Bypass Road, Zuarinagar, Sancoale, Goa 403726, India |
| Isuru Fernando | [@isuruf](https://github.com/isuruf) | [7](https://github.com/sympy/sympy/commits?author=isuruf) | [commits](https://github.com/sympy/sympy-paper/commits?author=isuruf), [comments](https://github.com/sympy/sympy-paper/search?q=commenter:isuruf&type=Issues) | University of Moratuwa, Bandaranayake Mawatha, Katubedda, Moratuwa 10400, Sri Lanka |
| Sumith Kulal | [@Sumith1896](https://github.com/Sumith1896) | [7](https://github.com/sympy/sympy/commits?author=Sumith1896) | [commits](https://github.com/sympy/sympy-paper/commits?author=Sumith1896), [comments](https://github.com/sympy/sympy-paper/search?q=commenter:Sumith1896&type=Issues) | Indian Institute of Technology Bombay, Powai, Mumbai, Maharashtra 400076, India |
| Robert Cimrman | [@rc](https://github.com/rc) | [1](https://github.com/sympy/sympy/commits?author=robert.cimrman@gmail.com) | [commits](https://github.com/sympy/sympy-paper/commits?author=rc), [comments](https://github.com/sympy/sympy-paper/search?q=commenter:rc&type=Issues) | New Technologies -- Research Centre, University of West Bohemia, Univerzitní 8, 306 14 Plzeň, Czech Republic |
| Anthony Scopatz | [@scopatz](https://github.com/scopatz) | [15](https://github.com/sympy/sympy/commits?author=scopatz) | [commits](https://github.com/sympy/sympy-paper/commits?author=scopatz), [comments](https://github.com/sympy/sympy-paper/search?q=commenter:scopatz&type=Issues) | University of South Carolina, Columbia, SC 29201 |

Note: This file was automatically generated from `authors.json` using `list_text.py`.

**Comment by certik** [comment]
Criterion 1. is satisfied, and when taking into account commits and comments to this repo, I think 2. is also satisfied. So all is good here.

**Comment by certik** [comment]
@aterrel you should add all your emails addresses into your github profile, so that this link:

https://github.com/sympy/sympy/commits?author=aterrel

that is included in `authors.md` shows all your commits, i.e. including the following:

https://github.com/sympy/sympy/commits?author=aterrel@uchicago.edu
https://github.com/sympy/sympy/commits?author=aterrel@enthought.com

I haven't figured out how to convince github to show all your commits using a single link (as opposed to the 3 links above), since github is not using `.mailmap`, unless you add these emails into your profile yourself...

**Comment by aterrel** [comment]
good fun, I've always wondered about that. Will try to add them

**Pull request merged**